libsignal-service/src/cipher.rs

```diff
@@ -78,12 +83,27 @@ where
             let plaintext = self.decrypt(&envelope).await?;
             let message =
                 crate::proto::DataMessage::decode(plaintext.data.as_slice())?;
-            Ok(Some(Content::from_body(message, plaintext.metadata)))
+            let content = Content::from_body(message, plaintext.metadata);
+            Ok(Some(content))
         } else if envelope.content.is_some() {
             let plaintext = self.decrypt(&envelope).await?;
             let message =
                 crate::proto::Content::decode(plaintext.data.as_slice())?;
-            Ok(Content::from_proto(message, plaintext.metadata))
+            if let Some(bytes) = message.sender_key_distribution_message {
+                dbg!("PROCESSING SENDER KEYSSSSSSSS -------------------------");
```

Comment:
Clippy says it too, but this should probably go away. If you want, use `log::debug!()` etc. :-)

Reply:
the joys of `printf` debugging